optimism/contact/test/test_TwoBodyContact.py

```diff
@@ -28,11 +28,11 @@ def get_best_overlap_vector(mesh, edge, listOfEdges):
 
 # coarse search
 
-def test_contact_search(self):
-    sideA = self.mesh.sideSets['right1']
-    sideB = self.mesh.sideSets['left2']
-
-    closeEdgesinAForSidesInB = get_close_edge_list(sideA, sideB, self.mesh, self.disp, 0.15)
+#def test_contact_search(self):
+#    sideA = self.mesh.sideSets['right1']
+#    sideB = self.mesh.sideSets['left2']
+#
+#    closeEdgesinAForSidesInB = get_close_edge_list(sideA, sideB, self.mesh, self.disp, 0.15)
```

Comment:
Yup... pytest is pretty smart. It will run any method prefixed with the name "test_" in a file prefixed with "test_<other stuff>.py". It will also pick up unittest TestFixtures and run those too. It also works nicely with chex for jit/no jit testing that I've been starting to use on another project.

Anyway... I ripped out this dead code.